roles/ansible-deploy-rudder/library/rudder_node_settings.py

```diff
@@ -0,0 +1,544 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright: (c) 2021, Normation
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (
+    absolute_import,
+    division,
+    print_function
+)
+from ansible.module_utils.basic import AnsibleModule
+import json
+
+# import logging
+
+# Check if 'requests' is installed
+# Use : 'pip3 install requests' if not
+try:
+    import requests
+    from requests.packages.urllib3.exceptions import InsecureRequestWarning
+except ImportError:
+    HAS_REQUESTS = False
+else:
+    HAS_REQUESTS = True
+
+__metaclass__ = type
+
+# {{{ for debug (remove this)
+# logging.basicConfig(
+#     filename="/var/log/rudder/ansible/ansible_debug.log",  # tail -f -n0 /var/log/rudder/ansible/ansible_debug.log
+#     format='[%(asctime)s %(levelname)s %(module)s - %(funcName)s: %(message)s',
+#     datefmt='%Y-%m-%d %H:%M:%S]',
+#     encoding='utf-8',
```

Comment:
encoding fails on python3

Reply:
This part will be modified when adding the debugging functionality.